doc/api/child_process.md

```diff
@@ -261,15 +261,15 @@ lsExample();
 ```
 
 If the `signal` option is enabled, calling `.abort()` on the corresponding
-`AbortController` is similar to calling `.kill()` on the child process except
-the error passed to the callback will be an `AbortError`:
+`AbortController` is similar to calling `.kill()` on the child process except a
+different error is passed:
 
 ```js
 const { exec } = require('child_process');
 const controller = new AbortController();
 const { signal } = controller;
 const child = exec('grep ssh', { signal }, (error) => {
-  console.log(error); // an AbortError
+  console.log('aborted', signal.aborted); // true
```

Comment:
If find the comment confusing. Can we change this?
```suggestion
  console.log('aborted', signal.aborted); // aborted true
```
or
```suggestion
  console.log(signal.aborted); // true
```
or
```suggestion
  console.log('aborted');
  console.log(signal.aborted); // true
```

Reply:
Sure, this was just an initial pitch. I'd probably go with your first suggestion (which I also considered).

The second is cleaner, but makes less sense in real code, since there is no context to the logged value.

Reply:
I'd prefer option 2, doc examples priority is clarity, it's OK if it doesn't have to look like "real" code (one could argue that "real" code should not use `console.log` at all). It seems to me that the rest of the docs tend to not have "context" with the `console.log` calls.